virtcontainers/container.go

```diff
@@ -362,12 +363,28 @@ func (c *Container) mountSharedDirMounts(hostSharedDir, guestSharedDir string) (
 		filename := fmt.Sprintf("%s-%s-%s", c.id, hex.EncodeToString(randBytes), filepath.Base(m.Destination))
 		mountDest := filepath.Join(hostSharedDir, c.sandbox.id, filename)
 
-		if err := bindMount(m.Source, mountDest, false); err != nil {
-			return nil, err
+		var ephiVol = false
+		splitSourceSlice := strings.Split(m.Source, "/")
+		if len(splitSourceSlice) > 2 {
```

Comment:
Can you clarify the expected format of the original Source str in a comment?

Reply:
@egernst A typical ephemeral storage path looks like, `/var/lib/kubelet/pods/366c3a75-4869-11e8-b479-507b9ddd5ce4/volumes/kubernetes.io~empty-dir/cache-volume`

I will update the code with this sample path in code comments.